docs/navigate.md

```diff
@@ -57,6 +60,21 @@ update itself while the user is panning around on it (set to null to disable
 intermediate pans, the plot will then not update until the mouse button is
 released).
 
+**mode** a string specifies which pan mode to be use. Accepted values:
+empty string for normal mode without pan hint or direction snapping;
+'smart' for smart mode with pan hint. The graph movement will snap to one direction
+when the drag direction is close to it;
+'smart lock' for smart mode with pan hint. The graph movement will always snap to a direction
+that the drag path start with.
+Default: 'smart'.
+
+**enableTouch** enables the touch support, including pan (to drag), pinch (to zoom and move),
+and double tap (to recenter).
+
+**touchMode** a string specifies the pan mode of touch pan. Accepted values: empty string and 'smart'.
```

Comment:
Why do we have a separate 'touchMode' property for the touch case? Can't the touch plugin leverage both the 'smart' and 'smartLock' concepts?

Reply:
The root cause for touchMode property is the default navigation behavior is **different** for mouse drag (smart) and touch pan (normal). It has to be a separate property for touch mode to set a different mode for individual interaction.

Or, we can force both interactions to use the same pan mode, while it will change the default behavior and become a breaking change.

On the other hand, I will align the accepted values for mode and touchMode in the following commit.